src/main/java/overlap_use_case/TimetableModelUnpacker.java

```diff
@@ -0,0 +1,95 @@
+package overlap_use_case;
+
+import entities.*;
+import retrieve_timetable_use_case.*;
+
+import java.util.ArrayList;
+
+/** A class responsible for converting between Timetable Model classes (that is, including its internal
+ * model classes) back into their entity forms. "Unpacks the model class" into the entity.
+ */
+public class TimetableModelUnpacker {
+
+    public static Session unpackSessionModel(SessionModel sessionModel){
```

Comment:
I think it would be appropriate to write some javadoc for the methods in the TimetableModelUnpacker, since every non-trivial method should have some sort of documentation.

Reply:
Tried it! Strange since this is just syntax sugar (it seems like a valid expression, at least in the version of Java we're using!). 
Hopefully this more conventional version holds :D.

Reply:
Also done! :)